org-ql.el

```diff
@@ -938,10 +950,20 @@ Arguments STRING, POS, FILL, and LEVEL are according to
               :action (lambda ()
                         (throw 'found t)))))))))
 
+(org-ql--defpred parent (querypredicatefn)
+  "Return non-nil if the current entry's parent matches the compiled QUERYPREDICATEFN."
```

Comment:
I think the fact that the predicate will be byte-compiled is an implementation detail that shouldn't matter to this function.

Reply:
Ah, you're right. Updated the docstring.